tests/default/controllers/test_suite_supervisor/test_suite_supervisor.py

```diff
@@ -239,8 +239,10 @@ def run(self):
             receiver = self.getDevice("ts_receiver")
             receiver.enable(basicTimeStep)
 
-        # 30 seconds before executing the next world
-        timeout = time.time() + 30
+        # 30 seconds before executing the next world, 60 seconds for the robot_window_html test
+        delay = 60 if self.currentSimulationFilename.endswith('/robot_window_html.wbt') else 30
+        print(f'delay = {delay}')
```

Comment:
Is the printing needed or only for debugging?

Reply:
Oops, yes, I should remove it...